catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -449,4 +453,29 @@ public DeleteResponse process(DeleteResponse deleteResponse) throws PluginExecut
         modified, effective, expiration, created
     }
 
+    public void pingSubscriptionResponseHandlers() {
+
+        //// TODO (RWY) - make pool size configurable possibly move pool to constructor and make final.
+        ForkJoinPool forkJoinPool = new ForkJoinPool(4);
+        try {
+            forkJoinPool.submit(() -> {
+                try {
+                    bundleContext.getServiceReferences(Subscription.class, null)
+                            .parallelStream()
+                            .map(bundleContext::getService)
+                            .map(Subscription::checkActive)
+                            .collect(Collectors.toList());
+                } catch (InvalidSyntaxException e) {
+                    LOGGER.error("Error pinging subscription response handlers", e);
+                }
+            })
+                    .get();
```

Comment:
This looks weird on this line alone. Has this file been run through the code formatter?